src/app/(landing)/@faq/faq.module.scss

```diff
@@ -0,0 +1,35 @@
+@use '../../../styles/colors';
+
+.faq {
+  margin-top: 10rem;
+}
+
+.buttonsContainer {
+  margin: 48px 0;
+}
+
+.buttons {
+  background-color: #00948c; // idk how to use the color vars
+  border-radius: 32px;
+
+  button {
+
+    &:not(:first-child) {
+      margin-left: 48px;
+    }
+  }
+}
+
+.questions {
+  max-width: 872px;
```

Comment:
We should use rem instead of px and we are use the flex components over applying flex properties via scss